docs/src/understand/api-client-perspective/swagger.md

```diff
@@ -12,9 +19,33 @@ looking for, double-check the old.
 
 ## New docs
 
-These docs show swagger 2.0:
-
-[new staging swagger page](https://staging-nginz-https.zinfra.io/api/swagger-ui/)
+The Swagger documentation for public endpoints depends on the API version.
+Though, the version prefix in URL paths works for internal (private) endpoints,
+too, they are not versioned.
```

Comment:
Isn't the internal API just as versioned, and we merely don't make use of it yet, ie. all versions happen to look exactly alike?

Reply:
:thinking: 

I'll figure that out.